packages/chronos/chronos-lib/src/main/java/com/daimler/data/application/config/SwaggerConfig.java

```diff
@@ -82,7 +82,7 @@ public Docket api() {
 				.build();
 
 		RequestParameter authParamBuilder = new RequestParameterBuilder().name("Authorization")
-				.description("Authorization header").query(q -> q.defaultValue(defaultAuthToken))
```

Comment:
change the value in application.yml rather

Reply:
or since we are giving variables key values in vault, we can change there, no need to change in code here. Please revert this change